aptos-move/aptos-sdk-builder/src/common.rs

```diff
@@ -64,27 +64,6 @@ fn quote_parameter_as_field(arg: &ArgumentABI) -> Named<Format> {
 }
 
 pub(crate) fn make_abi_enum_container(abis: &[ScriptABI]) -> ContainerFormat {
-    let mut variants = BTreeMap::new();
-    for (index, abi) in abis.iter().enumerate() {
-        let mut fields = Vec::new();
-        for ty_arg in abi.ty_args() {
-            fields.push(quote_type_parameter_as_field(ty_arg));
-        }
-        for arg in abi.args() {
-            fields.push(quote_parameter_as_field(arg));
-        }
-        variants.insert(
-            index as u32,
-            Named {
-                name: abi.name().to_camel_case(),
-                value: VariantFormat::Struct(fields),
-            },
-        );
-    }
-    ContainerFormat::Enum(variants)
-}
-
-pub(crate) fn make_namespaced_abi_enum_container(abis: &[ScriptABI]) -> ContainerFormat {
```

Comment:
what is namespace?

Reply:
lol, I was creating distinct "namespaces" for functions by their module name. The original version of this code required distinct function names even for different modules. I think this bit you back in the day when coding move entry points.

Reply:
why this is deleted? it default to have namespaced?

Reply:
yeah, my code was so beautiful that it worked for both scriptfunctions and scripts (that are not "namespaced"). As I said, there's so much duplicate code in these generators  -- even I contributed :)